lib/integration_whitelist.rb

```diff
@@ -0,0 +1,113 @@
+INTEGRATION_WHITELIST = [
+  'active_directory',
+  'activemq',
+  'activemq_xml',
+  'agent_metrics',
+  'apache',
+  'aspdotnet',
+  'btrfs',
+  'cacti',
+  'cassandra',
+  'cassandra_nodetool',
+  'ceph',
+  'cisco_aci',
+  'cockroachdb',
+  'consul',
+  'coredns',
+  'couch',
+  'couchbase',
+  'crio',
+  # 'datadog_checks_base',   namespacing package for wheels (NOT AN INTEGRATION)
+  # 'datadog_checks_dev',      developer tooling for working on integrations (NOT AN INTEGRATION)
+  'directory',
+  'disk',
+  'dns_check',
+  'docker_daemon',
+  'dotnetclr',
+  'ecs_fargate',
+  'elastic',
+  'envoy',
+  'etcd',
+  'exchange_server',
+  'fluentd',
+  'gearmand',
+  'gitlab',
+  'gitlab_runner',
+  'go-metro',
+  'go_expvar',
+  'gunicorn',
+  'haproxy',
+  'hdfs_datanode',
+  'hdfs_namenode',
+  'http_check',
+  'hyperv',
```

Comment:
Has this been shipped to A5 already? If not let's remove it as well and include it only in A6

Reply:
Hyperv is indeed in the 6.9.x branch and wasn't a part of the omnibus blacklist, so it should have been shipped already.